.changeset/swift-flowers-invent.md

```diff
@@ -0,0 +1,5 @@
+---
+"@atproto/xrpc-server": patch
+---
+
+Make `params` and `input` available during auth
```

Comment:
Just a reminder to update this in case we do change the availability of `input`.  Sounds like that was a small open question.